src/main/java/uk/gov/hmcts/cmc/claimstore/documents/ClaimContentProvider.java

```diff
@@ -59,6 +60,8 @@ public ClaimContentProvider(
 
         map.put("claim", claimDataContentProvider.createContent(claim));
         map.put("responseDeadline", formatDate(claim.getResponseDeadline()));
+        LocalDateTime ccjIssuedAt = claim.getCountyCourtJudgmentIssuedAt().orElse(null);
```

Comment:
I think something like this would be cleaner:
```
claim.getCountyCourtJudgmentIssuedAt()
  .ifPresent(issuedAt -> map.put(formatDate(issuedAt));
```